app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -1,10 +1,11 @@
 #
-# Uses the AWS Config service to monitor for events.
+# Uses the AWS Config or CloudWatch service to monitor for events.
```

Comment:
@bronaghs is actually working on automated setup. But yeah, you need to configure it, then both AWSConfig and ClouWatch can send event into the Topic we have there. But in reality, that costs money, you are fine with one of them. :-)